### ISSUE TITLE
Just checking in if we can get this on Linux as yet?

### ISSUE THREAD

**Issue by therealrobster** [serge-rgb/milton#171]
Hi all,

I'm sure I used this in the past on linux but can't find anything here now.  Maybe I was mistaken?

Anyway, is there an end user friendly way to get Milton on Linux?  Would love it for an upcoming project.

Thanks so much

**Comment by serge-rgb** [comment]
I think there's an Arch package but I'm not familiar with it. I personally haven't built Milton on linux in a long time. Maybe the arch package is your best bet? Quick search shows this: https://aur.archlinux.org/packages/milton/
Hope this works!

**Comment by therealrobster** [comment]
> I think there's an Arch package but I'm not familiar with it. I personally haven't built Milton on linux in a long time. Maybe the arch package is your best bet? Quick search shows this: https://aur.archlinux.org/packages/milton/
> Hope this works!

Thanks for that.  I had a look but it does seem quite technical.  It'd be great to get it into the main repositories to get people using it easily.  Just hit the app store on your preferred distro and press install.  That would be amazing.  Thank for the link though, and the software.  Just wish I could use it :-)

**Comment by FaFre** [comment]
@therealrobster there is a package called `pamac`, which is already preinstalled in some Arch-based distros. You don't feel a difference between AUR/official repositories anymore from a usability perspective. Check that out if your scared from manual building ;)

BTW: I use the AUR package and it's working great!

**Comment by Pshemas** [comment]
yeah, would love it to see it on Linux as well, preferably as appimage or flatpak so it could be used in any distro of choice. Anyone seen a guide on howto build it on Linux?
